cdap-docs/developers-manual/source/building-blocks/namespaces.rst

```diff
@@ -28,8 +28,9 @@ namespaces include:
 The first version of namespaces was introduced in :ref:`CDAP v2.8.0 <release-notes>`, and
 is part of the :ref:`HTTP RESTful API v3 <http-restful-api-v3>`.
 
-Namespace Rules
-===============
+
+Namespace Components
+====================
 
 A Namespace consists of a namespace identifier (the namespace ID), a display name, and a description.
```

Comment:
I think this should be 'A namespace has a ...'

Primarily because when 'consists' could be misleading right - it could mean 'containership' in which case a namespace would consist of apps, data, etc.